ui/shared/components/panel/view-pedigree-image/LazyPedigreeImagePanel.jsx

```diff
@@ -0,0 +1,379 @@
+import React from 'react'
```

Comment:
this is just a renamed file, I don't know why git is not picking that up, but it doesn't need review